src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -28,11 +38,9 @@
     private final UserService userService;
 
     @ApiOperation(
-            value = "Authenticate a username and password against Delius Identity (LDAP)",
-            authorizations = {@Authorization("ROLE_AUTH_DELIUS_LDAP")})
```

Comment:
These values were incorrect.  They don't make it into the swagger docs and are invalid:
```
/api/identifiers/{type}/{value}.get.security.0
Security requirements must match a security definition
Jump to line 5044
```
when viewing at http://editor.swagger.io/.  I did see https://github.com/springfox/springfox/issues/1655 as well, which looked quite good.